modules/common/css/lib/errors.scss

```diff
@@ -2,21 +2,23 @@
 @mixin wdc-form-validation($_type) {
   $_borderColor: $wdc-color-black-pepper-100 !default; // This should not be used, it is necessary for the @if / @else if to.
 
-  @if ($_type== 'error') {
+  @if ($_type=='error') {
```

Comment:
Is it possible to get rid of the prettier noise here? I want to make sure this diff is clean in case we need to go back and see it.

Reply:
@stephanerangaya I think you may have something in your IDE that's screwing with our prettier rules. I just had to re-save the file and it undid all of this using our default formatting rules.

@lychyi Just pushed a commit to revert these changes